frontend/src/components/layout/whatsnew/WhatsNewList.module.scss

```diff
@@ -4,7 +4,8 @@
 .empty-message {
   display: flex;
   flex-direction: column;
-  padding: $spacing-lg;
+  padding-top: $spacing-lg;
+  padding-bottom: $spacing-2xl;
```

Comment:
thought: I really like this approach to padding, rather than `padding: top x-axis bottom` shorthand. What are your thoughts on making this a [style rule](https://stylelint.io/user-guide/rules/list/declaration-block-no-shorthand-property-overrides/)?